src/labels/CustomLabels.labels

```diff
@@ -7328,6 +7328,182 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
+        <fullName>gseuChecklistItemDonationOrganizationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Institutional and corporate giving are an important source of nonprofit support. Track organizational giving by creating an Opportunity from an Account record.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Learn more about Organization Donations in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Enter Organization Donations</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemDonationOrganizationTitle</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Donation Organization in GS for End User</shortDescription>
+        <value>Enter a Donation from an Organization</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationDesc</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Description of checklist Item Track Donation in GS for End User</shortDescription>
+        <value>Campaigns let you to track your constituent outreach in Salesforce. Use the Primary Campaign field on Opportunities to better evaluate your fundraising efforts.</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationLinkLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Link label of checklist Item Track Donation in GS for End User</shortDescription>
+        <value>Learn more about Campaigns in Trailhead</value>
+    </labels>
+    <labels>
+        <fullName>gseuChecklistItemTrackDonationPriBtnLabel</fullName>
+        <categories>Get Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Primary button label of checklist Item Track Donation in GS for End User</shortDescription>
```

Comment:
I recommend changing this to: Primary button label of Track Donation checklist item in Get Started for End Users.

Reply:
@smarovich the max length for a short description is 80 and this message has 82

Reply:
OK, let's use: Primary button label of Track Donation checklist item in GetStarted for End User